bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -60,6 +61,7 @@ export default createContainer(() => {
     wasKicked: wasUserKicked(),
     isLoading: getLoading(),
     modal: getModal(),
+    getCaptionsStatus,
```

Comment:
Following the other props this should be something like:
`captionsStatus: getCaptionsStatus()`